introduction/architecture.md

```diff
@@ -111,6 +110,68 @@ while maintaining support for multiple indexes.
 For more on the motivation and design of TimescaleDB's adaptive space/time
 chunking, please see our [technical blog post][chunking].
 
+## TimescaleDB Clustering (PRIVATE BETA) [](timescaledb-clustering)
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives described above.
+In the multi-node topology, all nodes are TimescaleDB instances, but may play different roles
+depending on their responsibilities within a distributed database.
+
+### Node Types
+
+The two roles that nodes can take on are:
+1) Access node
+2) Data node
+
+A front-end access node accepts all requests from the client, distributes the requests and queries
```

Comment:
Hmmm.... I wonder if the distinction between front-end access nodes and backend data nodes might be confusing. Are there front-end data nodes? Is there anything special that needs to be done for front-end vs. backend nodes?

Reply:
ha, let's just remove any mention of front or back end.